src/EditorFeatures/CSharp/Highlighting/KeywordHighlighters/AsyncAwaitHighlighter.cs

```diff
@@ -10,9 +11,20 @@
 
 namespace Microsoft.CodeAnalysis.Editor.CSharp.KeywordHighlighting.KeywordHighlighters
 {
-    internal abstract class AbstractAsyncHighlighter<TNode> : AbstractKeywordHighlighter<TNode> where TNode : SyntaxNode
+    [ExportHighlighter(LanguageNames.CSharp)]
+    internal class AsyncAwaitHighlighter : AbstractKeywordHighlighter
     {
-        protected void HighlightRelatedKeywords(SyntaxNode node, List<TextSpan> spans)
+        protected override bool HighlightNode(SyntaxNode node)
+            => node.IsReturnableConstruct();
+
+        protected override IEnumerable<TextSpan> GetHighlightsForNode(SyntaxNode node, CancellationToken cancellationToken)
```

Comment:
CancellationToken is unused? Should it be removed or passed along?

Reply:
It's an override so I can't remove it. I looked at all implementations and it's used in `ConditionalPreprocessorHighlighter` & `RegionHighlighter`.

Reply:
Ah. OK.